### ISSUE TITLE
Update react-a11y-img-has-alt - area elements of image maps have alternate text

### ISSUE THREAD

**Issue by HamletDRC** [microsoft/tslint-microsoft-contrib#223]
area elements of image maps have alternate text - wcag2a, wcag111, section508, section508.22.a - From area-alt

**Comment by cthulhu-uu** [comment]
This is my first time in this repo and I'd like to contribute for Hacktoberfest. Can you please clarify what change is being requested? Thanks!

**Comment by JoshuaKGoldberg** [comment]
Awesome, welcome to the repo @cthulhu-uu! The more PRs the merrier.

I believe the issue is asking that the `<area>` element declared in [`<map>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map) elements should be validated similar to regular `<img>` tags, in that they must have an `alt` attribute.

This should cause a `react-a11y-image-has-alt` complaint:

```tsx
<map name="infographic">
    <area shape="poly" coords="7,35,49" href="/HTML" target="_blank" />
</map>
```

...but this should not _(because of the `alt`)_:

```tsx
<map name="infographic">
    <area shape="poly" coords="7,35,49" href="/HTML" target="_blank" alt="Happy face" />
</map>
```

/cc @HamletDRC in case I'm horribly wrong.

Is that the clarification you were looking for?

**Comment by cthulhu-uu** [comment]
@Chovin something you want to look at? 👀

**Comment by Chovin** [comment]
I'd love to take this on if that's alright. Thanks @cthulhu-uu! ❤️

**Comment by JoshuaKGoldberg** [comment]
☠️  *It's time!* ☠️ 

Per #876, this repository is no longer accepting feature pull requests. TSLint is being deprecated and we recommend you switch to https://typescript-eslint.io.

Thanks for open sourcing with us, everyone!
